src/status_im/ui/screens/screens.cljs

```diff
@@ -263,6 +264,10 @@
             ;;TODO custom
             :options   {:topBar {:visible false}}
             :component community/community}
+           {:name      :discover-communities
+            ;;TODO animated-header scroll behaviours
+            :options   {:topBar {:visible false}}
+            :component discover-communities/communities}
```

Comment:
Maybe better to add this screen to `status-mobile/src/status_im/navigation2/screens.cljs` files, so it will be easier to delete old screens when needed.